### ISSUE TITLE
Textual representation of children along with their attributes and text values

### ISSUE THREAD

**Issue by Tulakshana** [drmohundro/SWXMLHash#168]
Following is extracted from the xml I'm trying to serialized. I would like to retrieve the elements, attributes and text inside the element 'abody' in their textual format. What is the correct way to go about this?

```
<abody>
                    <anid>AN0113544370;vj901mar.16;2016Mar08.13:04;v2.2.500</anid>
                    <title id="AN0113544370-1">SOME TITLE </title>
                    <p>SOME TEXT</p>
                    <p>Some text 
                        <ulink href="some url">some linked text</ulink> some more text
                    <hd id="AN0113544370-2">EVENTS</hd>
                    <p>Some text</p>
                    <p>Some text</p>
                    <hd id="AN0113544370-3">LONDON</hd>
                    <p>Some text</p>
                    <p>Some text</p>
                    <p>Some text</p>
</abody>
```
I tried doing xml["abody"].element?.description. Then the string contains the 'abody' tag also. I only need its children.

Expected output,

```
                    <title id="AN0113544370-1">SOME TITLE </title>
                    <p>SOME TEXT</p>
                    <p>Some text 
                        <ulink href="some url">some linked text</ulink> some more text
                    <hd id="AN0113544370-2">EVENTS</hd>
                    <p>Some text</p>
                    <p>Some text</p>
                    <hd id="AN0113544370-3">LONDON</hd>
                    <p>Some text</p>
                    <p>Some text</p>
                    <p>Some text</p>
```

**Comment by drmohundro** [comment]
It sounds like what you're asking about is essentially an `innerXML` type of property that would be the string format. Is that correct? It might make sense to call it `innerDescription` or `childrenDescription` just to stay consistent with the existing `description` property, but I'm open to ideas.

Do you have any thoughts?

**Comment by Tulakshana** [comment]
@drmohundro,
Yes, you got it right. I'm looking at something like an innerXML type of property. For the moment my work around is as follows,

```
        let body = xml['abody']
        var html = ""
        for child in body.children  {
                html += (child.element?.description ?? "")
        }
```

**Comment by drmohundro** [comment]
FYI, I just published `4.7.0` to include this change.
